src/entity.js

```diff
@@ -706,22 +713,29 @@ Entity.setup = function(options) {
     // Bind table name
     var method = client[name].bind(client, options.tableName);
 
+    const report = (start, status) => {
+      status = `.${status}`;
```

Comment:
any reason we couldn't do <code>status = `${name}.${status}`</code> here and avoid the double concatenation in the monitor calls later?